### ISSUE TITLE
Error parsing relatively simple address

### ISSUE THREAD

**Issue by kganjam** [datamade/usaddress#144]
import usaddress
usaddress.tag('1807 Glenwood St. NE Palm Bay FL 32907')

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "c:\python27\lib\site-packages\usaddress\__init__.py", line 190, in tag
    raise RepeatedLabelError(address_string, parse(address_string), label)
usaddress.RepeatedLabelError:
ERROR: Unable to tag this string because more than one area of the string has the same label

ORIGINAL STRING:  1807 Glenwood St. NE Palm Bay FL 32907
PARSED TOKENS:    [(u'1807', 'AddressNumber'), (u'Glenwood', 'StreetName'), (u'St.', 'StreetNamePostType'), (u'NE', 'StreetNamePreDirectional'), (u'Palm', 'StreetName'), (u'Bay', 'PlaceName'), (u'FL', 'StateName'), (u'32907', 'ZipCode')]
UNCERTAIN LABEL:  StreetName

When this error is raised, it's likely that either (1) the string is not a valid person/corporation name or (2) some tokens were labeled incorrectly

**Comment by jeancochrane** [comment]
Hey @kganjam,

Thanks for filing this issue. We're bringing in a big new batch of training data in the next few days that should address this problem. I'll leave this issue open until the fix has been made, and send you an update explaining how the behavior has changed.

For now, if you still need to parse this data I'd recommend preprocessing any addresses that include a `StreetNamePostDirectional` like this one by appending a comma to the post-directional element. (For example, this address should change from `1807 Glenwood St. NE Palm Bay FL 32907` to `1807 Glenwood St. NE, Palm Bay FL 32907` before you run it through the parser.) Appending the comma seems to produce the correct behavior:

```python
In [4]: usaddress.tag('1807 Glenwood St. NE, Palm Bay FL 32907')

Out[4]:
(OrderedDict([('AddressNumber', '1807'),
              ('StreetName', 'Glenwood'),
              ('StreetNamePostType', 'St.'),
              ('StreetNamePostDirectional', 'NE'),
              ('PlaceName', 'Palm Bay'),
              ('StateName', 'FL'),
              ('ZipCode', '32907')]),
 'Street Address')

**Comment by jeancochrane** [comment]
Fixed in [v0.5.8.](https://github.com/datamade/usaddress/commit/407ef238903c9c65da652c288af5990dc412773a)

New behavior:

```python
In [5]: usaddress.tag(' 1807 Glenwood St. NE Palm Bay FL 32907')
Out[5]:
(OrderedDict([('AddressNumber', '1807'),
              ('StreetName', 'Glenwood'),
              ('StreetNamePostType', 'St.'),
              ('StreetNamePostDirectional', 'NE'),
              ('PlaceName', 'Palm Bay'),
              ('StateName', 'FL'),
              ('ZipCode', '32907')]),
 'Street Address')
```

Get the latest release via pip:

```bash
pip install --upgrade usaddress
```

**Comment by kganjam** [comment]
Thanks for the fix!
